homework/task-1.js

```diff
@@ -0,0 +1,5 @@
+
+
+for(var line ="#"; line.length <=7;line +="#"){
+    console.log(line)

```

Comment:
წერტილ-მძიმესაც მიაქციე ყურადღება
```suggestion
    console.log(line);

```